config/conf.json

```diff
@@ -144,6 +144,22 @@
                 "include_in_custom_gce" : true
             }
         },
+        {
+            "name" : "terra-aou-bioconductor-tidyverse-3.16.0",
+            "base_label" : "All of Us",
+            "tools" : [
+                "r"
+            ],
+            "packages" : {},
+            "version" : "3.16.0",
+            "automated_flags" : {
+                "include_in_ui" : false,
+                "generate_docs" : false,
+                "build" : true,
+                "include_in_custom_dataproc" : true,
+                "include_in_custom_gce" : true
```

Comment:
Those two should be false. @jdcanas  to confirm

Reply:
👍  I have no idea what they are used for.  I literally copied the section from the other AoU image 
https://github.com/DataBiosphere/terra-docker/blob/a313154bf0f26a1df5c3c5e61bb9c536016a75ed/config/conf.json#L130:L146

Reply:
Yeah, these should can false. This flag causes them to be cached in the VM image created for VMs used by Terra UI.

Reply:
Set them to `false`